docs/reference/modules/network.asciidoc

```diff
@@ -119,9 +123,10 @@ sorted by IPv4/IPv6 stack preference, then by reachability. If you set a
 address for node-to-node communication, you should explicitly set
 `network.publish_host`.
 
-Both of the above settings can be configured just like `network.host` -- they
-accept IP addresses, host names, and
-<<network-interface-values,special values>>.
+This setting accepts IP addresses, host names, and
+<<network-interface-values,special values>>, but does not accept arrays.
+It should be used only when `network.host` is an array, and should be set
```

Comment:
This isn't quite true either, it could be that `network.host` resolves to multiple addresses, or `network.bind_host` is involved, or various other strange environmental concerns.